File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/uploadImage/UploadImageResponse.java
package com.lattis.ellipse.data.network.model.response.uploadImage;

import com.google.gson.annotations.SerializedName;
import com.lattis.ellipse.data.network.model.response.AbstractApiResponse;

import java.util.Arrays;

/**
 * Created by Velo Labs Android on 06-04-2017.
 */

public class UploadImageResponse extends AbstractApiResponse {
    @SerializedName("payload")
    public UploadImageResultResponse resultResponse;

    public String uploadedUrl() {
        return resultResponse.getUploaded_url();
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/uploadimage/UploadImageResponse.kt
package com.lattis.data.entity.response.uploadimage

import com.google.gson.annotations.SerializedName
import com.lattis.data.entity.response.base.AbstractApiResponse

class UploadImageResponse : AbstractApiResponse() {
    @SerializedName("payload")
    var resultResponse: UploadImageResultResponse? = null
    fun uploadedUrl(): String? {
        return resultResponse!!.uploaded_url
    }
}